google/cloud/forseti/scanner/audit/external_project_access_rules_engine.py

```diff
@@ -262,6 +262,36 @@ def __init__(self, rule_name, rule_index, rules):
         self.rule_index = rule_index
         self.rules = rules
 
+    def _applies_to_user(self, user_email):
```

Comment:
I like these changes. Consider adding unit tests for the new methods instead of relying on the unit tests that test multiple methods.